configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
+# QUOTA
+- name: ratelimit_by_service_by_consumer
+  quota_params:
```

Comment:
is correspondence with template type inferred by parsing 'xxx_params' ?

Seems like it will lead to readability issues. Wouldn't it be better to nest each entry under their associated template type name

e.g.
quota:
  name:
  params:
...
list_check:
  name:
  params:
...

This comment applies equally to constructors as it does to types.